<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SquDialog</class>
 <widget class="QDialog" name="SquDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>594</width>
    <height>370</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>方波</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resourcefile.qrc">
    <normaloff>:/Icon/Icon/wave-4.png</normaloff>:/Icon/Icon/wave-4.png</iconset>
  </property>
  <widget class="QPushButton" name="StandBy_pushBtn">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>300</y>
     <width>82</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>-1</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    border: 1px solid #000000;
    border-radius: 5px;
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                   stop: 0 #dedede, stop: 0.5 #434343,
                                   stop: 0.51 #000000, stop: 1 #656a6d);
    color: #FFFFFF;
    font: bold 14px;
    min-width: 80px;
}

QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                   stop: 0 #cfcccc, stop: 0.5 #333232,
                                   stop: 0.51 #000000, stop: 1 #585c5f);
    color: #00CC00;
}
QPushButton:flat {
    border: none;
}</string>
   </property>
   <property name="text">
    <string>STANDBY</string>
   </property>
  </widget>
  <widget class="QPushButton" name="OutputOn_pushBtn">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>300</y>
     <width>82</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>-1</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    border: 1px solid #000000;
    border-radius: 5px;
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                   stop: 0 #dedede, stop: 0.5 #434343,
                                   stop: 0.51 #000000, stop: 1 #656a6d);
    color: #FFFFFF;
    font: bold 14px;
    min-width: 80px;
}

QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                   stop: 0 #cfcccc, stop: 0.5 #333232,
                                   stop: 0.51 #000000, stop: 1 #585c5f);
    color: #00CC00;
}
QPushButton:flat {
    border: none;
}</string>
   </property>
   <property name="text">
    <string>OUTPUTON</string>
   </property>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>10</y>
     <width>231</width>
     <height>101</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>211</width>
      <height>74</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QLabel" name="SquWaveNo_Label">
         <property name="minimumSize">
          <size>
           <width>70</width>
           <height>31</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>70</width>
           <height>31</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="text">
          <string>方波类型:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="SquWaveNo_Cmb">
         <property name="minimumSize">
          <size>
           <width>85</width>
           <height>31</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>85</width>
           <height>31</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>11</pointsize>
          </font>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="SquWaveOhm_Label">
         <property name="minimumSize">
          <size>
           <width>70</width>
           <height>31</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>70</width>
           <height>31</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="lineWidth">
          <number>2</number>
         </property>
         <property name="text">
          <string>阻抗选择:</string>
         </property>
         <property name="textFormat">
          <enum>Qt::AutoText</enum>
         </property>
         <property name="buddy">
          <cstring>DcWaveNo_Cmb</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="SquOhm_Cmb">
         <property name="minimumSize">
          <size>
           <width>85</width>
           <height>31</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>85</width>
           <height>31</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Andalus</family>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="currentText">
          <string/>
         </property>
         <property name="currentIndex">
          <number>-1</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QFrame" name="frame_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>118</y>
     <width>231</width>
     <height>161</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>219</width>
      <height>141</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="SquFreq_Label">
         <property name="minimumSize">
          <size>
           <width>71</width>
           <height>31</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>71</width>
           <height>31</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="text">
          <string>输出频率:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="SquFreq_Dsb">
         <property name="minimumSize">
          <size>
           <width>82</width>
           <height>31</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>82</width>
           <height>31</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="buttonSymbols">
          <enum>QAbstractSpinBox::NoButtons</enum>
         </property>
         <property name="decimals">
          <number>5</number>
         </property>
         <property name="maximum">
          <double>1000.000000000000000</double>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="SquFreqUnit_Cmb">
         <property name="minimumSize">
          <size>
           <width>50</width>
           <height>31</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>50</width>
           <height>31</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>11</pointsize>
          </font>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QDial" name="SquFreq_Dial"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QFrame" name="frame_3">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>10</y>
     <width>269</width>
     <height>271</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="SquAmp_Label">
          <property name="minimumSize">
           <size>
            <width>71</width>
            <height>31</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>71</width>
            <height>31</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="text">
           <string>输出幅度:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="SquAmp_Dsb">
          <property name="minimumSize">
           <size>
            <width>82</width>
            <height>31</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>82</width>
            <height>31</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="buttonSymbols">
           <enum>QAbstractSpinBox::NoButtons</enum>
          </property>
          <property name="decimals">
           <number>5</number>
          </property>
          <property name="maximum">
           <double>150.000000000000000</double>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="SquAmpUnit_Cmb">
          <property name="minimumSize">
           <size>
            <width>78</width>
            <height>31</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>78</width>
            <height>31</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>11</pointsize>
           </font>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLCDNumber" name="lcdNumber">
        <property name="lineWidth">
         <number>2</number>
        </property>
        <property name="smallDecimalPoint">
         <bool>true</bool>
        </property>
        <property name="segmentStyle">
         <enum>QLCDNumber::Outline</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDial" name="SquAmp_Dial"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="resourcefile.qrc"/>
 </resources>
 <connections/>
</ui>
